### DISCUSSION TITLE
Hi guys!

### DISCUSSION THREAD

**Discussion by intoinside** [c128lib/.github#3] [Team Posts]
Hello @c128lib/c128lib-dev-team how are you?
I hope you are well, I haven't heard from you for a few weeks and I would like to ask you if you still want to contribute to c128lib.
I don't want to rush you, nor do I want to force you into any activity, but I would like to at least try to establish an activity plan on the library for future developments.
I have developed the three main projects and I am about to reach a first milestone of stability and completeness. I would therefore like to understand, together with you, how to move forward.
If you want to continue, if you need time, if you prefer to carry on other projects, if you don't want to participate anymore, if you can't or don't feel like participating anymore, please let me know.
Feel free to reply here or by email if you prefer (raffaele.intorcia@gmail.com).

Have a nice day :-) (I used google translate, forgive me but I'm a bit cold and my brain works very slowly)

PS: of course, if you want, you can publicize c128lib wherever you like to make it more and more popular :-D

**Comment by johnpalermo** [comment]
Ciao Raffaele,

I’m still very interested. I have created the framework repository but have not added anything yet. I’m still working on ideas and some code before pushing it.

I’m also very interested in your repositories and reading carefully.

But, you’re correct. I just need some more time. This is a very busy season with family and work and in November I had almost no time to spend on this. Because of my line of work, the end of the year is the absolute busiest time.

Also, I will start to promote the repositories on the Commodore groups that I’m a member of. I see that C128 popularity is growing now, even in North America.

I’m always open to conversation also. 😊

**Comment by intoinside** [comment]
Hi John!
Glad to hear you're still interested! Take your time, as I said before I don't want to rush because there are no deadlines. I'm also carrying on this library for fun :-)
I advertised c128lib on many Italian Facebook groups dedicated to Commodore. Also, between Christmas and the new year, an article will be published in an online magazine called RetroMagazine (https://www.retromagazine.net/). I don't know if there will be a publication on the English version as well as the Italian one, but it's still a good opportunity. I wrote a small article for them in which I talk about various things, including the current state of the library and future evolutions, especially of the Framework project.

I hope to hear you soon, if you want to share some ideas and discuss, I'm available!

**Comment by johnpalermo** [comment]
Great.  I see that you've advertised on a few of the English Facebook groups that I'm also a member of.  I can share also to some other social media platforms.  

That's exciting news to know that you have an article to be published on RetroMagazine.  I'll look for it.  

Speak to you soon. :)

**Comment by johnpalermo** [comment]
I made a quick post on 2 reddit communities about c128lib. I will look for other opportunities as well.
